internal/pkg/cli/flag.go

```diff
@@ -132,6 +133,8 @@ Mutually exclusive with -%s, --%s`, imageFlagShort, imageFlag)
 	wkldTypeFlagDescription = fmt.Sprintf(`Type of job or svc to create. Must be one of:
 %s`, strings.Join(template.QuoteSliceFunc(manifest.WorkloadTypes), ", "))
 
+	clusterFlagDescription = fmt.Sprintf(`Optional. The ARN of the cluster to run the task in. 
```

Comment:
Can we update this to say that they could also pass the short name for the cluster so that it works similar to `run-task`?

Reply:
I wonder if there is an opportunity for us to simplify the `--default` and `--cluster` behavior 🤔 

@huanjani discovered that we can hide flags with cobra:
https://github.com/aws/copilot-cli/blob/7ccb3106fa940c478a45831bfb29768b76e9f6ce/internal/pkg/cli/pipeline_init.go#L685

I wonder if we should now hide `--default` and if `--cluster` is specified with no `--subnets` we can default to the using  default VPC subnets?

Reply:
(After an extensive search into my memory) I recalled that the `--default` flag was introduced in order to skip prompting for `app` and `env`. It is completely an UX thing and does nothing to the `ConfigRunner`.

Reply:
Will this fail to run if the cluster doesn't exist in the default vpc? I want to make sure we're defaulting to the correct thing. Can we check under the hood whether the cluster has copilot tags, then use the correct subnets automagically?